controllers/articles.controller.js

```diff
@@ -27,8 +34,15 @@ exports.getCommentsCon = (req, res, next) => {
 exports.postCommentsCon = (req, res, next) => {
     const comment = req.body
     const endpoint = req.params
+    const article_id = endpoint.article_id
+    const promises = [postCommentsMod(comment, article_id)]
+    if (endpoint){
+        promises.push(checkArticleExists(article_id))
```

Comment:
I can see what you've done here, but you shouldn't need to use the checkArticleExists for this task. If the article doesn't exist, PSQL will throw an error that you can catch in your middleware 👍